jekyll/_cci2_ja/troubleshoot-pipelines.adoc

```diff
@@ -17,4 +17,4 @@ contentTags:
 
 This page describes errors and troubleshooting methods for pipelines and scheduled pipelines.
 
-include::../_includes/snippets/troubleshoot/pipelines-troubleshoot-snip.adoc[]
\ No newline at end of file
+include::../_includes/snippets/troubleshoot/ja/pipelines-troubleshoot-snip.adoc[]
```

Comment:
@yuki-sakamoto-san I _think_ all snippets need to be linked to using the `ja` in the path. I added this one in Crowdin